### PR TITLE
Tendrils and ash walker tendrils are now anchored like goliaths

### PR DESCRIPTION
**Pull request by jughu**
hey guys i brought an nest for the chef and now theres murderous lizards everywhere

### CODE REVIEW

**Comment by lzimann** [comment]
Update the title to match what is being changed on the pr.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/necropolis_tendril.dm [hunk 1]
```diff
@@ -22,6 +22,7 @@
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 	maxbodytemp = INFINITY
+	anchored = TRUE
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/necropolis_tendril.dm]: this is inherited from `/mob/living/simple_animal/hostile/spawner` already, and thus does not fix shit all.

File: code/modules/ruins/objects_and_mobs/ash_walker_den.dm [hunk 2]
```diff
@@ -9,6 +9,7 @@
 	faction = list("ashwalker")
 	health = 200
 	maxHealth = 200
+	anchored = TRUE
```
Comment (ChangelingRain) [inline code/modules/ruins/objects_and_mobs/ash_walker_den.dm]: this is already inherited from this is inherited from `/mob/living/simple_animal/hostile/spawner`, and would also be inherited from `/mob/living/simple_animal/hostile/spawner/lavaland` with your existing change.

**Comment by KorPhaeron** [comment]
This PR does literally nothing, we don't need it open for 24 hours

The correct fix would be to not allow anchored mobs to be buckled

**Comment by jughu** [comment]
Then l o w e r t h e i s s u e c o u n t e r

**Pull request closed without merging**